app/jobs/dqt/update_trainee_job.rb

```diff
@@ -22,7 +22,7 @@ def perform(trainee)
   private
 
     def trainee_updatable?(trainee)
-      VALID_STATES.include?(trainee.state)
+      VALID_STATES.include?(trainee.reload.state)
```

Comment:
I'm sneaking this in to see if it removes some dead jobs caused by a race condition.